base/sysinfo.jl

```diff
@@ -117,7 +117,7 @@ function _cpu_summary(io::IO, cpu::Array{CPUinfo}, i, j)
     println(io)
 end
 
-function cpu_summary(io::IO=STDOUT, cpu::Array{CPUinfo}=cpu_info())
+function cpu_summary(io::IO=STDOUT, cpu::Vector{CPUinfo}=cpu_info())
```

Comment:
agreed, but let's leave it to a different PR as it seems outside the scope of this one :)